modules/tinymce/src/core/test/ts/browser/keyboard/HomeEndKeysTest.ts

```diff
@@ -57,7 +57,7 @@ describe('browser.tinymce.core.keyboard.HomeEndKeysTest', () => {
       it('move caret out and at the beginning of the element', () => {
```

Comment:
Not sure what the consensus is now that we switched to mocha, But should this be uppercase at the beginning of the sensense?

Reply:
Our current standard is to always start with the JIRA number, so that really negates the need for it to start with a capital. eg:
```
it('TINY-4612: move caret out and at the beginning of the element',
```

Reply:
updated